src/Generators/Microsoft.Gen.ComplianceReports/Emitter.cs

```diff
@@ -174,6 +174,10 @@ private void OutArray(string name, Action action)
 
     private void OutNameValue(string name, string value)
     {
+        value = value
```

Comment:
Why not use `JsonSerializer.Serialize(value)` ?

Reply:
IIRC Generators can't reference packages (e.g. `System.Text.Json`). @geeknoid, is that correct?

Reply:
And they target NS2.0, so that wouldn't be possible, ok.

Reply:
That's what I thought until a few minutes ago. Someone just sent me https://github.com/dotnet/roslyn/blob/main/docs/features/source-generators.cookbook.md#use-functionality-from-nuget-packages which shows the MSBuild magic to make this happen.

I'll create an issue to track upgrading to that approach.